crypto/err/err.c

```diff
@@ -92,17 +93,16 @@ static ERR_STRING_DATA ERR_str_reasons[] = {
     {ERR_R_DSA_LIB, "DSA lib"},
     {ERR_R_X509_LIB, "X509 lib"},
     {ERR_R_ASN1_LIB, "ASN1 lib"},
+    {ERR_R_CRYPTO_LIB, "crypto lib"},
```

Comment:
?

Reply:
I found `CRYPTO` misleading because `crypto` is not a letter-wise abbreviation, while most others, including `RSA`, are.

Reply:
And already now we do not write `LIB`, but `lib`, for the same reason.

Reply:
Whether it's an abbreviation or not doesn't matter.  There *is* the idea of a `CRYPTO` library (set of functions)
(although to be frank, we have "CRYPTO" functions that are named differently, so I can see the confusion with those)

Reply:
All right, reverted to the UPPERCASE names.